site/assets/scss/_component-examples.scss

```diff
@@ -272,7 +272,7 @@
   overflow: auto;
 }
 
-.scrollspy-example-2 {
```

Comment:
@julien-deramond why do we want this change? :confused:

Reply:
I thought it was named `-2` because of its position in the examples. But that's not the case indeed. Reverted via https://github.com/twbs/bootstrap/pull/36195/commits/e2fda4ce002366dd86c6c45383081091cef4a474